src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,90 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { evaluateOnce } from '../derived/evaluateOnce';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  let alwaysNode;
+  let isStarted = false;
+  let isDone = false;
+  let wasStopped = false;
+  return {
+    start: animationCallback => {
+      if (isStarted) {
+        animationCallback && animationCallback({ finished: false });
+        return;
+      }
+      if (isDone) {
+        console.warn('Animation has been finished before');
+        // inconsistent with React Native
+        return;
+      }
+      isStarted = true;
+      evaluateOnce(
+        set(currentState.position, value),
+        currentState.position,
+        () => {
+          alwaysNode = always(
+            set(
+              value,
+              block([
+                cond(clockRunning(newClock), 0, startClock(newClock)),
+                node(newClock, currentState, config),
+                cond(currentState.finished, [
+                  call([], () => {
+                    isStarted = false;
+                    isDone = true;
+                    value.__setAnimation(null, !wasStopped);
+                    if (!wasStopped) {
+                      wasStopped = false;
+                    }
+                  }),
+                  stopClock(newClock),
+                ]),
+                currentState.position,
+              ])
+            )
+          );
+          value.__setAnimation({
+            node: alwaysNode,
```

Comment:
this seems a bit weird that we crate that object here. It makes it difficult to track who should control calling the callback. Would be easier to follow if we were passing node and callback as two arguments. That would indicate we expect `value` to call the callback.

Alternatively we should just pass the whole "animation" object to the `value` – the object that has `start` and `stop` methods + we can make it export `alwaysNode` for the sake of `value`s access and make it controll calling `animationCallback` when `value` or someone else calls into `stop`